client/app/startup/ClientApp.jsx

```diff
@@ -1,17 +1,20 @@
 import React from 'react';
 import { Provider } from 'react-redux';
+import Router from 'react-router';
+import createHistory from 'history/lib/createBrowserHistory';
 
 import createStore from '../stores/commentsStore';
-import CommentScreen from '../components/CommentScreen';
+import routes from '../routes/routes';
 
 const App = props => {
   const store = createStore(props);
-  const reactComponent = (
+  const history = createHistory();
+
+  return (
     <Provider store={store}>
-      <CommentScreen />
+      <Router history={history} children={routes} />
```

Comment:
if we were not using redux, then we'd provide to the router some kind of middleware, with a prop called "createElement", which would bind the props with a component: https://github.com/rackt/react-router/blob/master/docs/API.md#createelementcomponent-props

```
<Router createElement={createElement} />

// default behavior
function createElement(Component, props) {
  // make sure you pass all the props in!
  return <Component {...props}/>
}
```